File: Leetcode/ARRAY/JAVA/Easy Problems/P_409.java
import java.util.*;

public class P_409 {
  public static int longestPalindrome(String s) {

    int length = 0;

    Set<Character> hashset = new HashSet<>();

    for (int i = 0; i < s.length(); i++) {
      if (!(hashset.contains(s.charAt(i)))) {
        hashset.remove(s.charAt(i));
        length += 2;
      } else {
        hashset.add(s.charAt(i));
      }
    }

    if (hashset.size() != 0) {
      length++;
    }

    return length;
  }
}

File: Leetcode (DSA)/ARRAY/JavaScript/Easy Problems/P_409.js

var longestPalindrome = function (s) {
    let arr = [];
    let length = 0;

    for (let i = 0; i < s.length ; i++) {
        if (arr.includes(s.charAt(i))) {
            arr.splice(i, 1);
            length += 2;
        } else {
            arr.push(s.charAt(i));
        }
    }
    
    if(!(arr.length == 0)){
        length += 1;
    }
    
    return length;

};